### ISSUE TITLE
How to use Iris transition properly?

### ISSUE THREAD

**Issue by subdan** [BowlerHatLLC/feathers#1728]
Look at the video. I use Iris transition to move from one screen to another. But I have an issue. Plus Button and List doesn't masked (Iris mask doesn't work). When Iris transition ends the Button and List disappears instantly. How to fix this issue?

https://drive.google.com/open?id=1wZYrsS0JU0ju8RmCveWq-GXkf7nERtvu

**Comment by joshtynjala** [comment]
The List has its own mask that might be conflicting with the mask used by Iris. Try setting clipContent to false on the List.

**Comment by subdan** [comment]
It doesn't help
`list.clipContent = false;`

**Comment by joshtynjala** [comment]
Based on the video, it looks like the mask is working on some places, but not others. Is that right? So I can assume that you enabled depthAndStencil in your AIR application descriptor?

**Comment by subdan** [comment]
Yes, masking working on some places. Yes, I enabled depthAndStencil.

**Comment by joshtynjala** [comment]
You might also need to set clipContent in other places too. For instance, if you're using PanelScreen, that also has a clipContent property.

**Comment by subdan** [comment]
I disabled clipContent for PanelScreen and it did not helps.

**Comment by joshtynjala** [comment]
Did you disable it on both the PanelScreen and the List at the same time?

**Comment by subdan** [comment]
Yes.

**Comment by joshtynjala** [comment]
Are you using any filters?

**Comment by subdan** [comment]
It is @marpies theme. I think yes.
